### ISSUE TITLE
babel-plugin-lodash error, "no such file or directory [..]/node_modules/lodash"

### ISSUE THREAD

**Issue by chrisblossom** [gajus/pragmatist#8]
Guessing this has to do with npm@3.

```
node --version
v4.2.2

npm --version
3.5.0

npm run test

> write-file-webpack-plugin@1.0.0 test /Users/chris/github/write-file-webpack-plugin
> npm run pragmatist test


> write-file-webpack-plugin@1.0.0 pragmatist /Users/chris/github/write-file-webpack-plugin
> node ./node_modules/.bin/pragmatist "test"

[09:55:21] Starting 'pragmatist:task-pre-copy-clean'...
[09:55:21] Finished 'pragmatist:task-pre-copy-clean' after 8.4 ms
[09:55:21] Starting 'pragmatist:test-copy'...
[09:55:21] Finished 'pragmatist:test-copy' after 19 ms
[09:55:21] Starting 'pragmatist:test-build'...

events.js:141
      throw er; // Unhandled 'error' event
      ^
Error: ENOENT: no such file or directory, scandir '/Users/chris/github/write-file-webpack-plugin/node_modules/pragmatist/node_modules/babel-plugin-lodash/node_modules/lodash'
    at Error (native)
    at Object.fs.readdirSync (fs.js:808:18)
    at getDirectories (/Users/chris/github/write-file-webpack-plugin/node_modules/pragmatist/node_modules/babel-plugin-lodash/lib/lodash-modules.js:24:23)
    at Object.<anonymous> (/Users/chris/github/write-file-webpack-plugin/node_modules/pragmatist/node_modules/babel-plugin-lodash/lib/lodash-modules.js:34:42)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/chris/github/write-file-webpack-plugin/node_modules/pragmatist/node_modules/babel-plugin-lodash/lib/index.js:135:22)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)

npm ERR! Darwin 15.0.0
npm ERR! argv "/Users/chris/n/bin/node" "/Users/chris/n/bin/npm" "run" "pragmatist" "test"
npm ERR! node v4.2.2
npm ERR! npm  v3.5.0
npm ERR! code ELIFECYCLE
npm ERR! write-file-webpack-plugin@1.0.0 pragmatist: `node ./node_modules/.bin/pragmatist "test"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the write-file-webpack-plugin@1.0.0 pragmatist script 'node ./node_modules/.bin/pragmatist "test"'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the write-file-webpack-plugin package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./node_modules/.bin/pragmatist "test"
npm ERR! You can get their info via:
npm ERR!     npm owner ls write-file-webpack-plugin
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/chris/github/write-file-webpack-plugin/npm-debug.log

npm ERR! Darwin 15.0.0
npm ERR! argv "/Users/chris/n/bin/node" "/Users/chris/n/bin/npm" "run" "test"
npm ERR! node v4.2.2
npm ERR! npm  v3.5.0
npm ERR! code ELIFECYCLE
npm ERR! write-file-webpack-plugin@1.0.0 test: `npm run pragmatist test`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the write-file-webpack-plugin@1.0.0 test script 'npm run pragmatist test'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the write-file-webpack-plugin package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run pragmatist test
npm ERR! You can get their info via:
npm ERR!     npm owner ls write-file-webpack-plugin
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/chris/github/write-file-webpack-plugin/npm-debug.log
```

**Comment by gajus** [comment]
This tool is incredibly biased.

Specifically, the https://github.com/gajus/canonical integration, which sets the coding standards. One of the requirements is that the project must be using `lodash` package. The error that you are getting is complaining that it cannot find that package.

While I would love to see more people using `pragmatist`, I doubt that will be the case given very strict requirements.

On the bright side, it helped me immensely to standardise the code style and build process across many public and private projects, saving countless of hours with every day of use.

**Comment by gajus** [comment]
Note that `lodash` is required to meet the linting rules of https://github.com/wix/eslint-plugin-lodash3 that are declared in https://github.com/gajus/eslint-config-canonical.

**Comment by chrisblossom** [comment]
Thanks for the quick response. I just wanted to open an issue so you'd be aware of it. I found out by forking your [write-file-webpack-plugin](https://github.com/gajus/write-file-webpack-plugin). I solved it by specifying 1.1.x in the devDeps.
